### PR TITLE
feat(Text): add alignment convenience functions

### PR DESCRIPTION
**Pull request by Eeelco**
Added convenience functions left_aligned(), centered() and right_aligned() plus unit tests. No example code updated since none seems to be using Text::alignment().

### CODE REVIEW

**Comment by Eeelco** [comment]
PR 2/4 for #853

File: src/text/text.rs [hunk 1]
```diff
@@ -211,6 +211,8 @@ impl<'a> Text<'a> {
     /// Sets the alignment for this text.
     ///
     /// Defaults to: [`None`], meaning the alignment is determined by the rendering widget.
+    /// Setting the alignment of a Text generally overrides the alignment of its
+    /// parent Widget.
     ///
     /// Alignment can be set individually on each line to override this text's alignment.
```
Comment (Valentin271) [inline src/text/text.rs:217]: This line could also be reported on other shortcut methods.

**Review by Valentin271 — APPROVED** [review]
Looks good, my comment is optional.

**Comment by joshka** [comment]
You'll probably need to rebase this.

**Review by orhun — APPROVED** [review]

File: src/text/text.rs (hunk 1 above)
Comment (Eeelco) [reply src/text/text.rs:217]: I also added this to the new convenience functions, or which functions do you mean?

Comment (Eeelco) [reply src/text/text.rs:217]: Aah I see, thanks for the clarification! Added it to the other methods

**Comment by Valentin271** [comment]
I think you pushed modifications from your other PR (for line) because you made the changes on your main branch.
That's what I deduce looking at your main branch.

I recommend you update your fork - using the button in the screenshot below - and then rebase this branch. You should have a small conflict in `examples/list.rs`

image

**Comment by Eeelco** [comment]
> I think you pushed modifications from your other PR (for line) because you made the changes on your main branch. That's what I deduce looking at your main branch.
> 
> I recommend you update your fork - using the button in the screenshot below - and then rebase this branch. You should have a small conflict in `examples/list.rs`
> 
> image

I tried to rebase properly but only made it worse, so I just re-created the branch and force pushed, hope that's ok

**Comment by Valentin271** [comment]
> I tried to rebase properly but only made it worse, so I just re-created the branch and force pushed, hope that's ok

Yep perfectly fine, thanks!

**Pull request merged**